src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py

```diff
@@ -59,6 +61,33 @@ def dcos_browse(name, resource_group_name):
 
     return
 
+def dcos_install():
+    """
+    Downloads the dcos command line from Mesosphere
+    """
+    install_location = ""
+    file_url = ""
+
+    system = platform.system()
+    if system == 'Windows':
+        install_location = 'C:\Program Files\dcos.exe'
+        file_url = 'https://downloads.dcos.io/binaries/cli/windows/x86-64/dcos-1.8/dcos.exe'
+    elif system == 'Linux':
+        install_location = '/usr/local/bin/dcos'
+        file_url = 'https://downloads.dcos.io/binaries/cli/linux/x86-64/dcos-1.8/dcos'
+    elif system == 'Darwin':
+        install_location = '/usr/local/bin/dcos'
+        file_url = 'https://downloads.dcos.io/binaries/cli/darwin/x86-64/dcos-1.8/dcos'
+    else:
+        logger.error('unknown system: %s' % system)
+
+    user_location = input('dcos binary install location [%s]:' % install_location)
+    if len(user_location) == 0:
+        user_location = install_location
+
+    logger.info("Downloading client to %s\n" % user_location)
+    urllib.request.urlretrieve(file_url, user_location)
```

Comment:
Should wrap this in a try/except as it can raise an `IOError` if the connection cannot be made (https://docs.python.org/2/library/urllib.html#urllib.urlopen).

e.g.
```
try:
    urllib.request.urlretrieve(file_url, user_location)
except IOError:
    raise CLIError('Connection error attempting to download client')
```